card.go

```diff
@@ -94,22 +95,33 @@ func (c *Card) set(ks []string, v any) {
 				return
 			}
 		}
-		if v == nil {
-			delete(c.data, p)
-		} else {
-			c.data[p] = v
+
+		keys := strings.Split(p, ".")
+		lastKey := keys[len(keys)-1]
+		var x any = c.data
+		// Get the object even if nested.
+		for i := 0; i < len(keys)-1; i++ {
```

Comment:
I'd rewrite this as `for ... range` for better readability:

```go
for _, key := range keys {
  x = get(x, key)
}
```

Reply:
Sure, the problem is I need to ignore the last entry. Do you think this would be more readable?

```go
for _, key := range keys[:len(keys)-1] {
  x = get(x, key)
}
```